packages/api/src/brevo-config/entities/brevo-config-entity.factory.ts

```diff
@@ -23,19 +26,29 @@ export class BrevoConfigEntityFactory {
             implements: () => [DocumentInterface],
         })
         class BrevoConfig implements BrevoConfigInterface, DocumentInterface {
-            [OptionalProps]?: "createdAt" | "updatedAt";
+            [OptionalProps]?: "createdAt" | "updatedAt" | "isApiKeySet" | undefined;
 
             @PrimaryKey({ columnType: "uuid" })
             @Field(() => ID)
             id: string = v4();
 
-            @Property({ columnType: "text" })
-            @Field()
-            senderMail: string;
+            @Property({ columnType: "text", nullable: true })
+            @Field({ nullable: true })
+            senderMail?: string;
 
-            @Property({ columnType: "text" })
-            @Field()
-            senderName: string;
+            @Property({ columnType: "text", nullable: true })
+            @Field({ nullable: true })
+            senderName?: string;
+
+            @Property({ columnType: "number", nullable: true })
+            @Field(() => Int, { nullable: true })
+            doiTemplateId?: number;
+
+            @Property({ columnType: "text", nullable: true })
+            apiKey?: string;
+
+            @Field(() => Boolean)
+            isApiKeySet: boolean;
```

Comment:
I think this can be removed since it's determined in the field resolver and isn't actually set here